wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/cors/CorsConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package com.cognifide.aem.stubs.wiremock.cors;
+
+public class CorsConfiguration {
+  private final String allowHeaders;
+  private final String allowMethods;
+  private final String allowOrigin;
+  private final boolean enabled;
+
+  public CorsConfiguration(boolean enabled, String allowHeaders, String allowMethods, String allowOrigin) {
+    this.allowHeaders = allowHeaders;
+    this.allowMethods = allowMethods;
+    this.allowOrigin = allowOrigin;
+    this.enabled = enabled;
+  }
+
+  public static CorsConfiguration disabled() {
```

Comment:
then `WiremockServletConfig.corsDisabled()` but as you prefer

Reply:
see above